app/views/admin/products/_form.html.haml

```diff
@@ -5,5 +5,5 @@
     = f.hidden_field(:picture_cache)
     = f.file_field(:picture)
     = f.number_field :on_hand_count
-    = f.number_field :price, class: :text_field, step: :any
+    = f.number_field :amount, class: :text_field, step: :any
```

Comment:
step of 0.01 would make more sense. You wouldn't charge someone 7.0004323 for something

Reply:
@fiestacasey how can i actually set this? form_for won't allow me to enter in "0.01" or 0.01 for the step.

Reply:
@fiestacasey i've concluded that this is impossible and that i don't like form_for 👍

Reply:
@gsnovak Did you try removing the class from this? 
`= f.number_field :amount, step: 0.01`?

Reply:
i figured out what i was doing wrong. lets never speak of this again.